commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/ChatArgument.java

```diff
@@ -77,42 +65,7 @@ public CommandAPIArgumentType getArgumentType() {
 
 	@Override
 	public <CommandSourceStack> BaseComponent[] parseArgument(CommandContext<CommandSourceStack> cmdCtx, String key, CommandArguments previousArgs) throws CommandSyntaxException {
-		final CommandSender sender = CommandAPIBukkit.<CommandSourceStack>get().getCommandSenderFromCommandSource(cmdCtx.getSource()).getSource();
-		BaseComponent[] component = CommandAPIBukkit.<CommandSourceStack>get().getChat(cmdCtx, key);
-
-		Optional<PreviewableFunction<BaseComponent[]>> previewOptional = getPreview();
-		if (this.usePreview && previewOptional.isPresent() && sender instanceof Player player) {
-			try {
-				BaseComponent[] previewComponent = previewOptional.get()
-					.generatePreview(new PreviewInfo<>(new BukkitPlayer(player), CommandAPIHandler.getRawArgumentInput(cmdCtx, key), cmdCtx.getInput(), component));
-
-				component = previewComponent;
-			} catch (WrapperCommandSyntaxException e) {
-				throw e.getException();
-			}
-		}
+		final BaseComponent[] component = CommandAPIBukkit.<CommandSourceStack>get().getChat(cmdCtx, key);
 		return component;
```

Comment:
Should we inline this to match other arguments?
```suggestion
		return CommandAPIBukkit.<CommandSourceStack>get().getChat(cmdCtx, key);
```

Reply:
Yup.